Cargo.toml

```diff
@@ -9,6 +9,20 @@ readme = "README.md"
 keywords = ["adler32", "hash", "rolling"]
 license = "Zlib"
 
+[dependencies]
+# Internal features, only used when building as part of libstd, not part of the
+# stable interface of this crate.
+core = { version = '1.0.0', optional = true, package = 'rustc-std-workspace-core' }
```

Comment:
IIUC This needs the `rename-dependency` feature which makes the whole manifest unreadable until 1.31